### PR TITLE
reboot: update French translation, update halt description

### PR DESCRIPTION
**Pull request by noraj**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.fr/linux/reboot.md [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
+- Arrête (met fin à tous les processus et arrête le processeur) le système (identique à `halt`) :
 
 `reboot --halt`
 
-- Réinitialisez le système maintenant mais propre :
+- Redémarre immédiatement sans contacter le gestionnaire du système :
 
 `reboot --force`
+
+- Écrit l'entrée wtmp shutdown sans redémarrer le système :
```
Comment (nicokosi) [inline pages.fr/linux/reboot.md:22]:
```suggestion
- Écrit l'entrée wtmp `shutdown` sans redémarrer le système :
```

File: pages/linux/reboot.md [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 `reboot --poweroff`
 
-- Halt the system (same as `halt`):
+- Halt (terminates all processes and shuts down the cpu) the system (same as `halt`):
```
Comment (nicokosi) [inline pages/linux/reboot.md:14]:
```suggestion
- Halt (terminates all processes and shuts down the CPU) the system (same as `halt`):
```

**Review by nicokosi — APPROVED** [review]
Looks good to me, with minor optional remarks. 😇

**Review by kbdharun — APPROVED** [review]

**Pull request merged**